### PR TITLE
Prettier plugin

### PR DESCRIPTION
**Pull request by EugeneTseitlin**

### CODE REVIEW

**Review by overbit — CHANGES_REQUESTED** [review]

File: .amplicationconfig.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-    "repository": "public.ecr.aws/r7y2l7b7/dsg",
+    "repository": "amplication/data-service-generator",
     "tag": "latest"
   },
   "plugins": [
     {
-      "id": "db-mongo",
-      "distPath": "./plugins/db-mongo/dist"
+      "id": "prettier",
+      "distPath": "./plugins/prettier/dist"
```
[the remark this answers is not in the document]
Comment (EugeneTseitlin) [reply .amplicationconfig.json:9]: It's just an instruction for plugin test runner which plugins to use. We can remove config from repo and write an instruction how to use it. But in in the scope of this issue :)

File: plugins/prettier/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "my-plugin",
+  "version": "0.0.1",
+  "description": "Description of your plugin",
```
[the remark this answers is not in the document]
Comment (EugeneTseitlin) [reply plugins/prettier/package.json:4]: Done

File: plugins/prettier/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "main": "dist/index.js",
+  "scripts": {
+    "dev": "webpack --watch",
+    "build": "webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Amplication Contributor",
```
[the remark this answers is not in the document]
Comment (EugeneTseitlin) [reply plugins/prettier/package.json]: Done

File: plugins/prettier/package.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (EugeneTseitlin) [reply plugins/prettier/package.json]: Done

**Pull request merged**